pkg/cmd/provisioning/identity_provider.go

```diff
@@ -324,12 +454,12 @@ func NewIdentityProviderSetup() *cobra.Command {
 		Run: identityProviderCmd,
 	}
 
-	identityProviderSetupCmd.PersistentFlags().StringVar(&CreateOpts.InfraName, "infra-name", "", "Name prefix for all created AWS resources")
-	identityProviderSetupCmd.MarkPersistentFlagRequired("infra-name")
+	identityProviderSetupCmd.PersistentFlags().StringVar(&CreateOpts.NamePrefix, "name-prefix", "", "Name prefix for all created AWS resources")
+	identityProviderSetupCmd.MarkPersistentFlagRequired("name-prefix")
```

Comment:
This helps clear up it's not infra id :) Does the name prefix here have to match future executions of the command? I'm guessing not. As per below I'd just clarify that this is just to help users identify resources visually.

Reply:
If you run ccoctl with different name-prefix values, you'll end up with a second set of objects altogether..